packages/components/inputs/date-time-input/src/date-time-input.tsx

```diff
@@ -157,11 +202,11 @@ class DateTimeInput extends Component {
       (prevState) => ({
         calendarDate: today,
         highlightedIndex:
-          prevState.suggestedItems.length +
+          (prevState.suggestedItems as string[]).length +
```

Comment:
Just curious about why we need to explicit use the type here because we already typed the component state in its declaration.

Reply:
True, but, it could be undefined. I updated this to use an empty array if suggestedItems is undefined.